src/components/SingleMessage.jsx

```diff
@@ -0,0 +1,78 @@
+import { formatDistance } from 'date-fns'
+import './SingleMessage.css'
+
+export const SingleMessage = ({ singleMessage, fetchPosts }) => {
+
+    const onLikeIncrease = async (thoughts_id) => {
```

Comment:
You dont need to send the id as an argument as you have access to the id through the singleMessage-prop